cql2pgjson/src/test/java/org/folio/cql2pgjson/ForeignKeyGenerationTest.java

```diff
@@ -212,4 +212,16 @@ public void searchItemByHoldingsPermanentLocationId() throws Exception {
         + "WHERE permanentLocationId='53cf956f-c1df-410b-8bea-27f712cca7c0')";
     assertEquals(expected, sql);
   }
+
+
+    @Test
+  public void searchItemByHoldingsPermanentLocationName() throws Exception {
```

Comment:
This is a unit test only.
Please also add an integration test to https://github.com/folio-org/raml-module-builder/blob/master/cql2pgjson/src/test/java/org/folio/cql2pgjson/ForeignKeyGenerationIT.java that ensures that PostgreSQL can correctly process the generated SQL. Simply extend https://github.com/folio-org/raml-module-builder/blob/master/cql2pgjson/src/test/resources/foreignKey.sql as needed.

Reply:
done